### DISCUSSION TITLE
感觉雾凇那边的中英混输词库不错，有考虑参考添加嘛？

### DISCUSSION THREAD

**Discussion by expoli** [rimeinn/rime-moran#82] [Ideas]
https://github.com/expoli/rime-ice/blob/master/en_dicts/cn_en.txt

**Comment by ksqsf** [comment]
可以的，不过得处理一下（一个音节不应该只有一个字母，否则会出问题）

目前其实已经有一些混合词了：
https://github.com/ksqsf/rime-moran/blob/main/moran.words.dict.yaml#L7156-L7164

**Reply by ClixTW** [reply]
單字母的話直接重複感覺比較合理？比如`T恤 ttxu`、`U盤 uupj`，U盤的U要打`yb`個人覺得實在不太直覺…要加詞也麻煩，得一個個想讀音。也可能是我剛接觸自然碼，少見多怪了😅

**Reply by ksqsf** [reply]
模擬漢字發音只是我個人的想法，不怪自然碼 lol

直接重複字母有個問題是會擠佔「非雙拼組合」，如 tt 原本不是雙拼，那麼如 thtt塘 ynbb寅 之類的字就不會被切成 th'tt yn'bb 兩個音節，這樣打這些字就可以省掉 o 或 / 補碼。

不過你提到的這個問題我認爲有更好的解法！以前編碼沒有大寫字母是因爲當時還沒有把大寫字母增加到 alphabet 中，既然現在支持大寫字母，就可以直接用大寫字母編碼了：

<img width="531" alt="image" src="https://github.com/user-attachments/assets/41139e66-73bd-4ca0-a0bf-205f1efd6bf9" />

這部分修改已 push。

PS：「U盤」還有個簡碼 upj。

**Reply by ClixTW** [reply]
哦哦，果然事情沒有憨人想得那麼簡單～大寫字母編碼感覺是不錯的選擇，感謝超快速的更新！
